src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>To do list</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="hello">
+    <section class="to-do-app">
+      <h1 class="to-do-tittle">To Do list</h1>
+      <div class="todo-main-div">
+        <div class="to-do-header">
+          <h3>Today's To Do</h3>
+          <button class="refresh">
+            <i class="fa-solid fa-rotate fa-2x fa-border-none"></i>
+          </button>
+        </div>
+        <div class="to-do-body">
+          <form class="first-form" action="">
+            <input
+              type="text"
+              class="add-task"
+              placeholder="Add to your list..."
+            />
+            <button class="add-btn">Add</button>
+          </form>
+          <ul class="to-do-list">
```

Comment:
- [ ] Kindly note that when a project is required to be displayed dynamically, it means it should be rendered from the js file. Therefore, I will advise you to remove all the added html tags and create the data in an array like 

```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

Then you can iterate over it with a loop by doing something like 

```
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

### Note, it does not have to be like that, I just used that as an example of dynamic display or dynamic rendering.